build.gradle.kts

```diff
@@ -22,15 +22,17 @@ dependencies {
 
   implementation("org.flywaydb:flyway-core")
   implementation("io.hypersistence:hypersistence-utils-hibernate-60:3.3.2")
+  implementation("org.hibernate.orm:hibernate-community-dialects:6.2.3.Final")
+  implementation("org.hibernate.orm:hibernate-core:6.2.3.Final")
 
   implementation("javax.transaction:javax.transaction-api:1.3")
   implementation("javax.xml.bind:jaxb-api:2.3.1")
 
-  implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.0")
+  implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.1")
```

Comment:
iirc 2.15.1 is the version for spring boot 3.1, rather than spring boot 3.  Not sure it matters though 🤷

Reply:
Good spot - will remove if have any issues

Reply:
actually you should be able to remove the line completely, think it is included by other dependencies anyway